### PR TITLE
Pin app shell version to tool version

### PR DESCRIPTION
**Pull request by ibacher**
## Requirements

- [x] My work conforms to the [**OpenMRS 3.0 Styleguide**](https://om.rs/styleguide).
- [x] I checked for feature overlap with [**existing widgets**](https://om.rs/directory).

## Summary

Changes the `openmrs` tool's app-shell dependency to exactly match the version of the app shell.

## Screenshots

_None._

## Related Issue

https://issues.openmrs.org/browse/O3-967

## Other

_None._

### CODE REVIEW

File: packages/tooling/openmrs/package.json [hunk 1]
```diff
@@ -13,7 +13,8 @@
     "start": "node ./dist/cli.js",
     "test": "jest --passWithNoTests",
     "build": "tsc",
-    "lint": "eslint src --ext ts,tsx"
+    "lint": "eslint src --ext ts,tsx",
+    "version": "yarn add @openmrs/esm-app-shell@$npm_package_version --exact"
```
Comment (ibacher) [inline packages/tooling/openmrs/package.json:17]: There really has to be a better way to do this than this but...

Comment (brandones) [reply packages/tooling/openmrs/package.json:17]: What's this for?

Comment (brandones) [reply packages/tooling/openmrs/package.json:17]: Oh funny. I've opened https://github.com/lerna/lerna/issues/3046 .

**Review by brandones — APPROVED** [review]
Thanks @ibacher !

**Pull request merged**